lxqt-admin/lxqt-admin-time/lxqt-admin-time_ca.ts

```diff
@@ -21,16 +21,16 @@
     <message>
         <location filename="../../../datetime.ui" line="70"/>
         <source>Enable network time synchronization (NTP)</source>
-        <translation type="unfinished"></translation>
+        <translation>Habilita NTP (Network Time Protocol)</translation>
```

Comment:
You translated it as  “Enable NTP”, but it's more like “Enable time synchronization through NTP”.